docs/src/main/sphinx/develop/insert.rst

```diff
@@ -0,0 +1,30 @@
+=============================================
+Supporting ``INSERT`` and ``CREATE TABLE AS``
+=============================================
+
+To support ``INSERT``, a connector must implement:
+
+* ``beginInsert()`` and ``finishInsert()`` from the ``ConnectorMetadata``
+  interface;
+* a ``ConnectorPageSinkProvider`` that receives a table handle and returns
+  a ``ConnectorPageSink``.
+
+When executing an ``INSERT`` statement, the engine calls the ``beginInsert()``
+method in the connector, which receives a table handle and a list of columns.
+It should return a ``ConnectorInsertTableHandle``, that can carry any
+connector specific information and will be passed to the page sink provider.
+The ``PageSinkProvider`` creates a page sink, that should accept ``Page`` objects.
+
+When all the pages for a specific split have been processed, the Trino engine
+calls ``ConnectorPageSink.finish()``, which returns a ``Collection<Slice>``
+of "fragments" representing connector-specific information about
```

Comment:
drop the ""

Reply:
I copied this from the existing `delete and update` section. I fixed it there too, in a separate commit.